sdk/auth/cached_authorizer.go

```diff
@@ -67,6 +68,15 @@ func (c *CachedAuthorizer) AuxiliaryTokens(ctx context.Context, req *http.Reques
 	return c.auxTokens, nil
 }
 
+// ExpireTokens expires the currently cached token and auxTokens, forcing new tokens to be acquired when Token() or AuxiliaryTokens() are next called
+func (c *CachedAuthorizer) ExpireTokens() error {
+	c.token.Expiry = time.Now()
```

Comment:
sanity-checking: what if `c.Token` is nil here?

Reply:
thanks, good catch